src/mem/mmap/mmap_mmu.c

```diff
@@ -112,12 +122,19 @@ void mmap_free(struct emmap *mmap) {
 
 void mmap_clear(struct emmap *mmap) {
 	struct marea *marea;
+	struct phy_page *phy_page;
 
 	dlist_foreach_entry(marea, &mmap->marea_list, mmap_link) {
 		vmem_unmap_region(mmap->ctx, marea->start, mmu_size_align(marea->end - marea->start));
 
 		marea_destroy(marea);
 	}
+
+	dlist_foreach_entry(phy_page, &mmap->page_list, page_link) {
+		phymem_only_free(phy_page->page, phy_page->page_number);
```

Comment:
This is correct, but something similar should be performed in munmap to prevent double free in such code
```c
p = mmap(ANON)
munmap(p) // phymem_free called
exit(0) // here mmap_clear will try to free phymem again
```